src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1041,6 +1037,18 @@ public LocalDate fromString(String string) {
           checkBoxCvrTreatBlankAsUndeclaredWriteIn
               .setSelected(ContestConfig.SUGGESTED_TREAT_BLANK_AS_UNDECLARED_WRITE_IN);
         }
+        case CSV -> {
+          buttonAddCvrFile.setDisable(false);
+          textFieldCvrFilePath.setDisable(false);
+          buttonCvrFilePath.setDisable(false);
+          textFieldCvrContestId.setDisable(false);
```

Comment:
How is this used? It doesn't look like the reader has any concept of a CVR with multiple contests in it, so I'm not sure why the user would supply a contest ID.